### PR TITLE
stop using svg-jar

### PR DESCRIPTION
**Pull request by mansona**
svg-jar has some issues when consumed in a v2 addon, this PR will unblock us from merging https://github.com/ember-learn/guidemaker-ember-template/pull/188

### CODE REVIEW

File: addon/constants/icons.js
Comment (NullVoxPopuli) [inline addon/constants/icons.js:1]: why not a sprite sheet with `<use>`? (mostly benefits high repetition icon rendering)

Comment (mansona) [reply addon/constants/icons.js:1]: because I have no idea how 😂 PRs welcome

Comment (NullVoxPopuli) [reply addon/constants/icons.js:1]: This covers it: https://www.sitepoint.com/use-svg-image-sprites/

the tricky thing with an addon, is that the spitesheet / symbol map would need to be either its own component, or a public asset that would need to render once -- other wise the `<use>` things have nothing to reference.

I think there are some webpack / vite plugins that can generate this from from files tho -- so maybe it's best to leave to consuming app decisions idk.

atm, I don't know if embedding SVG in JS can be detected by packagers :thinking: 

I'll think about some PRs once we start moving the ember family of websites to vite :tada:

**Review by IgnaceMaes — APPROVED** [review]
:shipit:

**Pull request merged**